src/services/refactors/extractMethod.ts

```diff
@@ -0,0 +1,885 @@
+/// <reference path="../refactorProvider.ts" />
+/// <reference path="../../compiler/checker.ts" />
+
+/* @internal */
+namespace ts.refactor.extractMethod {

```

Comment:
Change the other one to match. Having the subnamespace makes it more clear what's going on from the unit test side